src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
-export interface Figure {
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
 
+export interface Figure {
+  shape: Shapes;
+  color: string;
+  getArea: () => number;
 }
 
 export class Triangle implements Figure {
+  public shape: Shapes;
```

Comment:
shape = Shapes.Triangle;
  You can use your enum in that way!)

Reply:
correct